cmd/create/cluster/cmd.go

```diff
@@ -1502,18 +1496,17 @@ func run(cmd *cobra.Command, _ []string) {
 	}
 
 	// Cluster-wide proxy configuration
-	enableProxy := args.enableProxy
+	enableProxy := false
 	httpProxy := args.httpProxy
 	httpsProxy := args.httpsProxy
 	additionalTrustBundleFile := args.additionalTrustBundleFile
-	if httpProxy != "" || httpsProxy != "" || additionalTrustBundleFile != "" {
-		enableProxy = true
-	}
+
 	if useExistingVPC && interactive.Enabled() {
```

Comment:
this should really be
```
if useExistingVPC && !enableProxy && interactive.Enabled() {
```

Reply:
not if we do not put back the 'if' check above. Please see my other comment.